ios/StripeApplePay.mm

```diff
@@ -8,17 +8,12 @@ @interface RCT_EXTERN_MODULE(StripeApplePay, NSObject)
                   rejecter: (RCTPromiseRejectBlock)reject)
 
 RCT_EXTERN_METHOD(
-                  isApplePaySupported: (RCTPromiseResolveBlock)resolve
+                  isPlatformPaySupported:(RCTPromiseResolveBlock)resolve
                   rejecter: (RCTPromiseRejectBlock)reject)
 
 RCT_EXTERN_METHOD(
-                  presentApplePay:(NSDictionary *)params
-                  clientSecret:(NSString *)clientSecret
-                  resolver:(RCTPromiseResolveBlock)resolve
-                  rejecter:(RCTPromiseRejectBlock)reject)
-
-RCT_EXTERN_METHOD(
-                  confirmApplePayPayment:(NSString *)clientSecret
+                  confirmPlatformPay:(NSString *)clientSecret
+                  params:(NSDictionary *)params
```

Comment:
More recent versions have merged `presentApplePay` and `confirmApplePay` into one action with the Platform Pay API, so updating to align with that